packages/shadcn/src/components/ui/separator.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import * as React from 'react';
+import * as SeparatorPrimitive from '@radix-ui/react-separator';
```

Comment:
Rather than use this, import each of the things you need directly from `react` and `@radix/react-separator`